presto-docs/src/main/sphinx/release/release-0.101.rst

```diff
@@ -58,3 +61,19 @@ Hive Changes
 * Add JMX stats to PrestoS3FileSystem.
 * Add ``hive.recursive-directories`` config option to recursively scan
   partition directories for data.
+
+
+SPI Changes
+-----------
+
+* Add connector callback for rollback of ``INSERT`` and ``CREATE TABLE AS``.
+* Introduce an abstraction for representing physical organizations of a table
+  and describe properties such as partitioning, grouping, predicate and columns.
```

Comment:
Ah, yes, I refactored the sentence a bit after I wrote the initial version and missed that